lib/metadata_presenter/version.rb

```diff
@@ -1,3 +1,3 @@
 module MetadataPresenter
-  VERSION = '0.26.1'.freeze
+  VERSION = '0.26.2'.freeze
```

Comment:
i think this has now gone to 0.27.0

Reply:
Looks like current version has already moved on to 0.27.1 so I've made it 0.27.2 for this PR.
See https://github.com/ministryofjustice/fb-metadata-presenter/pull/104/commits/fcf72fd0491cbe505a09b8c173aa7064a9535e6a